src/main/java/bio/terra/cli/command/helperclasses/BaseCommand.java

```diff
@@ -26,8 +27,11 @@
 
   @Override
   public Integer call() {
-    // read in the global and workspace context
+    // read in the global and setup logging
```

Comment:
Ultra-tiny nit, feel free to ignore: "global context" (or could just remove the comment altogether).

Reply:
Fixed.